app/models/expert.rb

```diff
@@ -2,4 +2,8 @@ class Expert < ApplicationRecord
   belongs_to :user
   has_many :expert_fields
   has_many :fields, through: :expert_fields
+  has_many :requests
+
+  validates :description, presence: true, length: { minimum: 25 }
+  validates :price_per_hour, presence: true, numericality: { only_integer: true }
```

Comment:
should user id presence be true as well? same comment for all other references in other table, their presence should always be true, no?